server/settings_test.go

```diff
@@ -41,6 +42,8 @@ func TestSettings(t *testing.T) {
 				bodySettings := res.Payload.Settings
 				res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
 					Body: server.ChangeSettingsBody{
+						DisableStt:      !bodySettings.SttEnabled,
```

Comment:
:+1: could you fix it for telemetry too, please

Reply:
done